api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -61,3 +64,18 @@ type KubeadmConfigList struct {
 func init() {
 	SchemeBuilder.Register(&KubeadmConfig{}, &KubeadmConfigList{})
 }
+
+// Files defines the input for generating write_files in cloud-init.
+type Files struct {
+	// Path specifies the full path on disk where to store the file.
+	Path string `json:"path"`
+
+	// Owner specifies the ownership of the file, e.g. "root:root".
+	Owner string `json:"owner"`
```

Comment:
Owner should likely also be optional, since it isn't required for cloud-init.

Reply:
Fixed, PTAL